test/e2e/autoscaling/horizontal_pod_autoscaling.go

```diff
@@ -65,6 +65,16 @@ var _ = framework.KubeDescribe("[HPA] Horizontal pod autoscaling (scale resource
 		})
 	})
 
+	framework.KubeDescribe("[Serial] [Slow] StatefulSet", func() {
```

Comment:
I also verified existence of scale subresource for apps/v1beta2 using `kubectl proxy`.